parser/TableCellNode.php

```diff
@@ -63,4 +63,21 @@ public function getColSpan()
     {
         return $this->data['attrs']['colspan'];
     }
+
+    /**
+     * Calculate the correct padding to align cell-content left, right or center
+     *
+     * @return String[] [left padding, right padding]
+     */
+    protected function calculateAlignmentPadding()
+    {
+        if ($this->data['attrs']['align'] === 'right') {
+            return ['  ', ' '];
+        }
+        if ($this->data['attrs']['align'] === 'center') {
+            return ['  ', '  '];
+        }
+        return [' ', ' '];
```

Comment:
Hi Michael,

There is difference between 'left align' and 'not specifying the alignment'. This becomes visible if you have rtl-languages. Of course this code is the to-syntax conversion, not sure if you make this assumption more often.

See also for example:
https://github.com/splitbrain/dokuwiki/issues/2169
https://github.com/splitbrain/dokuwiki/pull/2224

Reply:
Ah, I was wondering if there is a difference, but I only came up with the possibility of obscure templates that might center-align by default and didn't think of rtl-languages. Thanks for pointing that out!